src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -744,6 +755,19 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
                         />
                     )}
 
+                    {showContentInsightsToggle && (
+                        <>
+                            <div className="bdl-SharedLinkSection-separator" />
+                            <div className="shared-link-toggle-row">
```

Comment:
similar to the separator, I think we should avoid using a class that was intended for another component.

are there any specific styles we need from `shared-link-toggle-row`? it looks like it uses the top margin, any other styles we need?
maybe we could create a generic `bdl-UnifiedShareForm-row` or `bdl-UnifiedShareForm-toggle` that has only the bare styles.